### ISSUE TITLE
[🐛BUG] [simplex] is not the name of an existing model

### ISSUE THREAD

**Issue by Fabrice-Clement** [RUCAIBox/RecBole#1423]
**Describe the bug**

Thanks for sharing this RecSys framework !

I want to try the SimpleX model but it is not recognized.

I've also tried simplex, SIMPLEX...

Thanks in advance

**To Reproduce**

```
from recbole.quick_start import run_recbole
run_recbole(model='simplex', dataset='ml-100k')

```
**Desktop (please complete the following information):**
 - OS: linux
- RecBole Version : think it is 1.0 ( have done : it clone https://github.com/RUCAIBox/RecBole.git today )
 - Python Version : 3.8.5

**Comment by Sherry-XLL** [comment]
Hello, @Fabrice-Clement

The code of SimpleX is in the under-developed branch, `1.1.x`, which we will release this month. If you want to use it right now, please switch the branch to `1.1.x` and run again. 

Thanks for your attention to RecBole!

**Comment by Sherry-XLL** [comment]
@Fabrice-Clement Hello, we have released the [1.1.1 version of RecBole](https://github.com/RUCAIBox/RecBole/releases/tag/v1.1.1) on Oct 5, and SimpleX is a general model in the master branch now. You can refer to our [documentation](https://recbole.io/docs/user_guide/model/general/simplex.html) for more details. 

Thanks for your attention to RecBole, and feel free to ask questions by opening a new issue if you have any problems!
